test_current_setup.playbook.yaml

```diff
@@ -110,89 +118,234 @@
     command: "ip netns exec {{ gw_ns }} arp -s 192.168.10.{{ item }} {{ downlink_mac_2 }}"
     with_sequence: 2-10
 
+  - name: "create static arp entries for subnet1 (192.168.20.x addresses) hosts"
+    command: "ip netns exec {{ gw_ns }} arp -s 192.168.20.{{ item }} {{ downlink_mac_3 }}"
+    with_sequence: 2-10
+
+  - name: "create static arp entries for subnet2 (192.168.30.x addresses) hosts"
+    command: "ip netns exec {{ gw_ns }} arp -s 192.168.30.{{ item }} {{ downlink_mac_4 }}"
+    with_sequence: 2-10
+
   - name: "configuring {{ internet_ns }} with: {{ item }}"
     command: "ip netns exec {{ internet_ns }} {{ item }}"
     with_items:
       - "ip address add 210.0.0.100/24 dev veth_world"
       - "ip route add 192.168.0.0/24 via 210.0.0.1"
       - "ip route add 192.168.10.0/24 via 210.0.0.1"
+      - "ip route add 192.168.20.0/24 via 210.0.0.1"
+      - "ip route add 192.168.30.0/24 via 210.0.0.1"
+
 
   - name: "set {{ item }} up"
     command: "ip l set {{ item }} up"
     with_items:
       - "{{ gw_iface }}"
       - "{{ ovs_bridge_gw }}"
 
-  - name: "delete {{ outer_iface }}"
-    command: "ip link delete {{ outer_iface }}"
+  - name: "delete {{ outer_iface_1 }}"
+    command: "ip link delete {{ outer_iface_1 }}"
+    ignore_errors: yes
```

Comment:
these commands are getting out of hand, we should find some logic and move them into ansible blocks with iterators